.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: php
 php:
-  - 5.3
```

Comment:
this version was failing in travis as the OS running the tests no longer supports this version